ingest.py

```diff
@@ -21,9 +21,52 @@
 RECORD_MANAGER_DB_URL = os.environ["RECORD_MANAGER_DB_URL"]
 
 
+def _get_text(tag):
+    new_line_elements = {"h1", "h2", "h3", "h4", "code", "p", "li"}
+    code_elements = {"code"}
+    skip_elements = {"button"}
+    for child in tag.children:
+        if isinstance(child, Tag):
+            # if the tag is a block type tag then yield new lines before after
+            is_code_element = child.name in code_elements
+            is_block_element = is_code_element and "codeBlockLines_e6Vv" in child.get(
```

Comment:
is_block_element = is_code_block_element? 

to avoid conflation w/ https://www.w3schools.com/html/html_blocks.asp